### PR TITLE
[Themes 1676] Fixed triple-chain large-manual-promo Desktop view

### PR DESCRIPTION
**Pull request by malavikakoppula**
## Test Steps

1. Checkout this branch `git checkout THEMES-1676-fix`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/triple-chain-block`
3. Add large-manual-promo-blocks to triple-chain. 
4. Check for Desktop view, Large-manual-promo Image and description are side-by-side in desktop view

[THEMES-1676]: https://arcpublishing.atlassian.net/browse/THEMES-1676?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: blocks/triple-chain-block/themes/news.json [hunk 1]
```diff
@@ -29,6 +29,15 @@
 			},
 			"desktop": {}
 		}
+	},"triple-chain-child-item-grid": {
```
Comment (vgalatro) [inline blocks/triple-chain-block/themes/news.json]: There should be a new line between `},` and `"triple-chain-child-item-grid"`.

**Review by vgalatro — CHANGES_REQUESTED** [review]
@malavikakoppula , just one small formatting change, then this is all good to go!

File: blocks/triple-chain-block/themes/news.json (hunk 1 above)
Comment (malavikakoppula) [reply blocks/triple-chain-block/themes/news.json]: @vgalatro Added new line

**Review by vgalatro — APPROVED** [review]
LGTM

**Pull request merged**